main.go

```diff
@@ -44,6 +46,11 @@ import (
 func init() {
 	log.SetLevel("warn")
 
+	err := godotenv.Load()
```

Comment:
this will load it for all commands, not sure the implications this might have with the stack's env file. 

A second option is to only load it for `okteto up`, but that would give us a different experience than `okteto down` and `okteto exec` when parsing `okteto.yaml`, and I don't think that's a good idea.

Reply:
I would be explicit as we do for okteto stacks:
https://okteto.com/docs/reference/stacks#env_file-string-optional